cfgov/unprocessed/css/organisms/ask-search.less

```
@@ -3,6 +3,15 @@
    Ask Search organism
    ========================================================================== */
 
-.input-with-btn__bar .input-with-btn_btn button {
-    height: 35px;
-}
+// TODO: Reconcile this with the input with button in Capital Framework.
+.o-form__input-w-btn_btn-container {
+
+    // Only attach the button to the input at tablet/desktop sizes.
+    .respond-to-min( 480px, {
+        border-left: 0;
+        & .a-btn {
```

Trivial: The ampersand here isn't necessary, I don't think. And I think we usually put a line between standard properties and nested selectors.

```suggestion

        .a-btn {
```

---

I'll remove it since this block is so small, but we may want to favor including it for clarity. I just got tripped up with https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/pages/ask.less#L239, because I didn't pick up right away that it was actually `.ask-cfpb-page__answer .o-search-bar_input { … }` and not `.o-search-bar_input { … }`. The indent should show that, but when every rule viewable in a window is at the same indent it's not as clear. If we included `&` it would always be clear there's a placeholder for the parent class.

---

That's a good point!